src/commons/services/utils.js

```diff
@@ -108,6 +108,35 @@
             return clusterName;
         }
 
+        vm.getVolumeDetails = function(clusterId) {
+            volumeList = [];
+            clusterData = vm.clusterData.clusters;
+            len = clusterData.length;
+            for ( i = 0; i < len; i++ ) {
+                clusterObj = clusterData[i];
+                for( key in clusterObj ) {
```

Comment:
in top level object have only 2 properties .
1) node_id 
2) stats

Example:
{

    "8b0c637d-f5f7-40f6-acd9-50d5feb5f4b3":{
        "tendrl_context":{},
        "maps":{},
        "pools":{}
    },
    "stats":{}

}

and checking condition based on tendrl_context will come after top level object properties 
i means - Object.<node-id>.tendrl_context ...  stats and tendrl_context is not in same level.. can not be possible to check condition here for tendrl_context . Please see the response format properly.